test/e2e/e2e-test/nginx/src/test/java/org/apache/skywalking/e2e/DataAssertITCase.java

```diff
@@ -64,6 +65,10 @@ public void verify() throws IOException, InterruptedException {
             Assert.assertEquals(200, response.getStatusLine().getStatusCode());
         }
 
+        try (CloseableHttpResponse response = client.execute(new HttpGet(suffixEntry))) {
+            Assert.assertEquals(200, response.getStatusLine().getStatusCode());
```

Comment:
e2e is not checking through `Assert`. As you add a skip rule, the way to check is, after verifying traces exist, check this new endpoint's trace doesn't exist, or it doesn't exist in endpoint list through CLI.

Reply:
OK, we don't use the latest e2e infra tool here. We actually should apply query for this endpoint to real OAP backend, then verify it is an empty list as result.